extensions/m3o/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "m3o",
+  "title": "M3O",
+  "description": "Explore, discover and consume public APIs as simpler programmable building blocks.",
+  "icon": "command-icon.png",
+  "author": "andreaselia",
+  "categories": [
+    "Data",
+    "Web"
+  ],
+  "license": "MIT",
+  "commands": [{
+    "name": "avatar",
+    "title": "Avatar",
+    "subtitle": "M3O",
+    "description": "Generate an avatar.",
+    "mode": "view"
+  }, {
+    "name": "dns",
+    "title": "DNS",
+    "subtitle": "M3O",
+    "description": "DNS over HTTPS (DoH).",
```

Comment:
could we be more explicit about what this command actually does?

Reply:
Updated the description.